Console/Display.hs

```diff
@@ -184,7 +207,18 @@ progressStart pbar = do
     showBar pbar
     return ()
 
--- | Tick an element on the progress bar
+-- | Ticks an element on the given progress bar. Should be used within a
+-- progress bar update function that is passed into `progress`.
+--
+-- @
+-- progressFunction :: ProgressBar -> IO String
+-- progressFunction bar = do
+--   threadDelay 100000
+--   progressTick bar
+--   threadDelay 200000
+--   progressTick bar
+--   return "Completed!"
```

Comment:
Yep, noting what happens when the progression reaches the max would be a good idea. I'll also document what happens in cases where the max is exceeded, or the update function completes before the max is reached.